### DISCUSSION TITLE
one question about gf180/aes-hybrid design

### DISCUSSION THREAD

**Discussion by inv1s10n** [The-OpenROAD-Project/OpenROAD#6087] [Q&A]
I counted the changes in the number of cells with two different row heights during all place stages and found that after resizing, the number of cells with two different row heights was unbalanced. However, after dp, the number of cells with two different row heights became balanced. Can DP also do resizing? I didn't see any related content in the introduction of openDP

**Comment by maliberty** [comment]
DP does not resize so there is something off in your analysis.

**Reply by inv1s10n** [reply]
![Snipaste_2024-11-03_11-10-10](https://github.com/user-attachments/assets/bc575580-1c32-4a6a-83df-a2c2e6bfcc74)
I used a Python script to read the def file of the place stage, and counted the units of two different heights. The information obtained is shown in the figure.
After resizing, the quantity of 7t is much greater than that of 9t, but it is balanced after dp

**Reply by maliberty** [reply]
I thought you meant the command. The dp stage contains the balance_row_usage command

**Reply by inv1s10n** [reply]
![11111](https://github.com/user-attachments/assets/ca030e25-0b8f-464a-8269-45e27b42b323)
Thanks very much. but this command seems to have no introduction.
